.make.versions

```diff
@@ -23,5 +23,5 @@ CODE_QUALITY_VERSION=0.3.0
 DOC_QUALITY_VERSION=0.3.0
 INGEST_TO_PARQUET_VERSION=0.3.0
 
-KFP_DOCKER_VERSION=0.2.0
+KFP_DOCKER_VERSION=0.1.1
```

Comment:
why did you returned the major version back?

Reply:
because we are really on 0.1.1. Trying to be consistent